src/Config/Auth.php

```diff
@@ -127,6 +127,17 @@ class Auth extends BaseConfig
         'tokens',
     ];
 
+    /**
+     * --------------------------------------------------------------------
+     * Login validation rules
+     * --------------------------------------------------------------------
+     * Returns the rules that should be used for user login validation.
+     */
+    public array $loginRules = [
```

Comment:
It would be kind if there is `username` as a comment.
```
        // 'username' => 'required|max_length[30]|alpha_numeric_space|min_length[3]',
```